server/routes/induction/create/index.ts

```diff
@@ -89,5 +92,12 @@ export default (router: Router, services: Services) => {
     router.get('/prisoners/:prisonNumber/create-induction/has-worked-before', [
       workedBeforeCreateController.getWorkedBeforeView,
     ])
+
+    router.get('/prisoners/:prisonNumber/create-induction/reasons-not-to-get-work', [
+      asyncMiddleware(reasonsNotToGetWorkCreateController.getReasonsNotToGetWorkView),
```

Comment:
What does the `asyncMiddleware` do? And why do we only need it here / not need it on any of the other routes/handlers?